iothub/device/src/DeviceClient.cs

```diff
@@ -538,7 +582,7 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
 
         /// <summary>
         /// Registers a new delegate for the connection status changed callback. If a delegate is already associated,
-        /// it will be replaced with the new delegate. Note that this callback will never be called if the client is configured to use HTTP as that protocol is stateless
+        /// it will be replaced with the new delegate. Note that this callback will never be called if the client is configured to use HTTP, as that protocol is stateless.
```

Comment:
minor: "I" capital

Reply:
for "...it will be replaced with the new delegate"? It is a continuation from the line above.

Reply:
Aha didn't realize that. It appeared as a new line